vllm/forward_context.py

```diff
@@ -82,15 +82,15 @@ def set_forward_context(attn_metadata: Any,
                                          dtype=torch.int32)
         from vllm.distributed.parallel_state import get_dp_group
         dist.all_reduce(num_tokens_tensor, group=get_dp_group().cpu_group)
-        num_tokens_across_dp = num_tokens_tensor.tolist()
+        cumsum_tokens_across_dp = torch.cumsum(num_tokens_tensor, dim=0)
 
     global _forward_context
     prev_context = _forward_context
     _forward_context = ForwardContext(
-        attn_layers=vllm_config.compilation_config.static_forward_context,
+        smuggled_layers=vllm_config.compilation_config.static_forward_context,
```

Comment:
we can change `vllm_config.compilation_config.static_forward_context` to `vllm_config.compilation_config.static_forward_context.attn_layers` and `vllm_config.compilation_config.static_forward_context.moe_layers` to be specific. `smuggled_layers` sounds quite confusing.